librz/analysis/p/analysis_bf.c

```diff
@@ -274,22 +274,22 @@ RzPVector *bf_rlimit(RzILVM *vm, BfContext *ctx, ut64 id, ut64 addr) {
 	RzILOp *branch = rz_il_new_op(RZIL_OP_BRANCH);
 	RzILOp *goto_ = rz_il_new_op(RZIL_OP_GOTO);
 	RzILOp *perform = rz_il_new_op(RZIL_OP_PERFORM);
-	RzILOp *inv = rz_il_new_op(RZIL_OP_INV);
```

Comment:
Why this change? It's defined as `inv` in the original: https://github.com/BinaryAnalysisPlatform/bap/blob/master/lib/bap_core_theory/bap_core_theory_definition.ml#L49
```ocaml

module type Bool = sig
  val b0 : bool
  val b1 : bool
  val inv : bool -> bool
  val and_ : bool -> bool -> bool
  val or_ : bool -> bool -> bool
end
```

Reply:
well, the normal op boolean name is not, not inv

Reply:
Well, I know. But I think it's better to stick to the original, as @thestr4ng3r noted too.

Reply:
i disagree. on bap, names have been choosen sue the limitation of the lang, this is why you have `not_` or `and_` etc..

Reply:
i'm ok to map them to the original bap names via comments. but the name of the structure, etc.. has to be clear in C

Reply:
I am fine with the new names as long as there corresponding comments supplied. BOOLNOT looks weird though

Reply:
not really, we have logical not and boolean not. in C the difference is `ut32 x = 0x1234` 
*Boolean Not* is `!x` meanwhile *Logical Not* is `~x`

Reply:
Yes, I understand the difference, just complaining about naming.

Reply:
it follows the names for logical and/or/xor

Reply:
I think we should keep it as `inv`. Our source of reference is bap and not what may be perceived as more standard.